master/getting-started/bare-metal/bare-metal.md

```diff
@@ -42,7 +42,7 @@ You can apply host endpoint policies to three types of traffic:
 same host.
 
 Set the `applyOnForward` flag to `true` to apply a policy to forwarded traffic.
-See [policy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
+See [GlobalNetworkPolicy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
```

Comment:
Shouldn't this link to `globalnetworkpolicy#spec` and not `networkpolicy#spec`.

Reply:
ah good find! I'll fix it